index.html

```diff
@@ -13,7 +15,218 @@
+          </p>
+        </div>
+        <div id="category-content-item-2" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/workshop.png" alt="">
+          </div>
+          <h3>Roll With the Punches</h3>
+          <p class="summary">
+            <span>Meaning:</span> To tolerate or endure through the unexpected mishappenings you may encounter from time
+            to time.
+          </p>
+        </div>
+        <div id="category-content-item-3" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/lecture.png" alt="">
+          </div>
+          <h3>Down To The Wire</h3>
+          <p class="summary">
+            <span>Meaning:</span> A tense situation where the outcome is decided only in the last few seconds.
+          </p>
+        </div>
+        <div id="category-content-item-4" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/ignite.png" alt="">
+          </div>
+          <h3>A Lemon</h3>
+          <p class="summary">
+            <span>Meaning:</span> Usually a vehicle (but any device) that you buy that's defective.
+          </p>
+        </div>
+        <div id="category-content-item-5" class="content-item content-item-with-hover">
+          <div class="img">
+            <img src="assets/Icons/forum.png" alt="">
+          </div>
+          <h3>Love Birds</h3>
+          <p class="summary">
+            <span>Meaning:</span> A pair of people who have a shared love for each other.
+          </p>
+        </div>
+      </div>
+
+      <div class="categories-btn">
+        <button type="button">See More</button>
+      </div>
+
+    </section>
+
+    <section id="featured-persons" class="wrapper features">
+      <div id="featured-persons-max" class="internal">2</div>
+      <div class="title">
+        <h2>these persons do not exist</h2>
+        <div class="divider"></div>
+      </div>
+      <div class="content">
+
+        <div id="featured-persons-content-item-internal" class="content-item internal">
+          <div class="img">
+            <img class="decorator-img" src="assets/decorators/transparent.jpeg" alt="">
+            <img id="featured-persons-pic-internal" class="main-img" src="assets/persons/avatar-internal.png" alt="">
+          </div>
+          <div class="body">
+            <h3 id="featured-persons-name-internal">-name-</h3>
+            <h4 id="featured-persons-email-internal">-email-</h4>
+            <div class="divider"></div>
+            <p id="featured-persons-Sentence-internal">
+              -Sentence-
+            </p>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="more-btn">
+        <button type="button">
+          I wanna see more persons
+          <i class="fa-solid fa-chevron-down more-btn-chevron-down"></i>
+        </button>
+      </div>
```

Comment:
- [ ] Kindly make sure the all cards are aligned with each other.

![image](https://user-images.githubusercontent.com/35535130/181384893-3c2b5fe5-cdf5-42c4-aeb5-d244aaf0f0e1.png)